test/jobs/inat_import_job_test.rb

```diff
@@ -472,6 +473,14 @@ def test_import_job_create_prov_name
                  proposed_name_notes)
 
     assert(obs.sequences.one?, "Obs should have one sequence")
+
+    email = QueuedEmail.first
+    expected_subject = "#{@user.login} created #{name.text_name}"
+    assert_equal(
```

Comment:
I put the test in InatImportJobTest because

- it was the importer which exposed the bug
- it was easy -- this method is creating a name
- I didn't want to take the time to figure out another place to put this.